templates/base.html

```diff
@@ -61,12 +61,12 @@
         integrity="sha384-w1Q4orYjBQndcko6MimVbzY0tgp4pWB4lZ7lr30WKz0vr/aWKhXdBNmNb5D92v7s" crossorigin="anonymous">
     </script>
 {% block js %}
+{% if messages %}

```

Comment:
Good catch here, placing the `<script>` inside of the `{% if-block %}` as well!

Reply:
Thanks, Tim! :)